si/orange-canvas.yaml

```diff
@@ -0,0 +1,3536 @@
+        Inputs:<ul>{inputs}</ul><hr/>: Vhodi:<ul>{{inputs}}</ul><hr/>
+        Outputs:<ul>{outputs}</ul>: Izhodi:<ul>{{outputs}}</ul>
+        No inputs<hr/>: Ni vhodov<hr/>
+        No outputs: Ni izhodov<hr/>
+        'ul { margin-top: 1px; margin-bottom: 1px; }': false
+canvas/items/utils.py:
+    T: false
+    A: false
+    B: false
+    C: false
+    def `linspace`:
+        Count must be non-negative: false
+document/__init__.py:
+    quickmenu: false
+    schemeedit: false
+document/commands.py:
+    class `UndoCommand`:
+        def `__getstate__`:
+            _UndoCommand__initialized: false
+            _UndoCommand__text: false
+            _UndoCommand__children: false
+        def `__setstate__`:
+            _UndoCommand__initialized: false
+            _UndoCommand__text: false
+            _UndoCommand__parent: false
+            _UndoCommand__child_states: false
+    class `AddNodeCommand`:
+        def `__init__`:
+            Add %s: false
+    class `RemoveNodeCommand`:
+        def `__init__`:
+            Remove %s: false
+    class `AddLinkCommand`:
+        def `__init__`:
+            Add link: false
+    class `RemoveLinkCommand`:
+        def `__init__`:
+            Remove link: false
+    class `InsertNodeCommand`:
+        def `__init__`:
+            Insert widget into link: false
+    class `AddAnnotationCommand`:
+        def `__init__`:
+            Add annotation: false
+    class `RemoveAnnotationCommand`:
+        def `__init__`:
+            Remove annotation: false
+    class `MoveNodeCommand`:
+        def `__init__`:
+            Move: false
+    class `ResizeCommand`:
+        def `__init__`:
+            Resize: false
+    class `ArrowChangeCommand`:
+        def `__init__`:
+            Move arrow: false
+    class `AnnotationGeometryChange`:
+        def `__init__`:
+            Change Annotation Geometry: false
+    class `RenameNodeCommand`:
+        def `__init__`:
+            Rename: false
+    class `TextChangeCommand`:
+        def `__init__`:
+            Change text: false
+    class `SetAttrCommand`:
+        def `__init__`:
+            Set %r: false
+    class `SetWindowGroupPresets`:
+        def `__init__`:
+            Scheme: false
+            Scheme.WindowGroup: false
+            text: false
+            Set Window Presets: false
+document/editlinksdialog.py:
+    class `EditLinksDialog`:
+        def `__setupUi`:
+            Clear All: false
+    _Link: false
+    output: false
+    input: false
+    lineItem: false
+    class `LinksEditWidget`:
+        def `addLink`:
+            '%r is not an output channel of %r': false
+            '%r is not an input channel of %r': false
+        def `removeLink`:
+            No such link {0.name!r} -> {1.name!r}.: false
+        def `__updateState`:
+            <center><b>{0}<b></center>: false
+        def `__updateAnchorState`:
+            Click and drag to connect widgets!: Kliknite in vlecite, da povežete gradnike
```

Comment:
A je to dokončen dogovor - vikanje uporabnikov? 
Ko sem sama uporabnica orodij, pripomočkov, ki jih raziskujem sama, mi dajejo boljši občutek nagovori, ki so manj formalni. Zdi se mi, da je bolj prijetno, če "program" domneva, da se z njim bodem sama. In me tika.

Reply:
To sem tehtal in tehtal, ampak sem se odločil za vikanje.

Ko so v Ljubljani odprli IKEO, so bili vsi napisi s tikanjem. To mi je izpadlo tako bedno, da sem jim pisal, pa so mi odgovorili nekaj v slogu sproščenost pa to. Vprašal sem, a to pomeni, da me bodo tudi prodajalci sproščeno tikali in so zagotovili, da to pa ne. Ampak da ima IKEA pač to prakso. Potem pa sem šel raziskovat, kaj počnejo v različnih jezikih in bilo je nekako pol-pol. Z drugimi besedami: lažejo. Odvisno je od kulture države in par slovenskih kul sproščenih tipov se je odločilo, da se pri nas tikamo. Predpostavljam, da so to taisti ljudje, ki blagajničarki v Mercatorju rečejo "ej, dej mi še ene Marlboro".

Tolkien je v nekem pismu napisal, da orki veliko preklinjajo, ker bi radi izražali močna čustva, vendar tega niso zmožni povedati na inteligentnejši način kot s preklinjanjem. Podobno je s tem IKEA bi lahko napisala besedilo, ki bi bilo sproščeno ... ker tega ne znajo, pa se zatekajo k cenenim trikom - tikanju.

Skratka, meni je všeč, če se s prodajalci, zobozdravniki, sprevodniki in snažilkamo vikamo, zato raje ne bi prispeval h "kulturi (cenene) sproščenosti". Ni pa bila trenutna odločitev - sem kar razmišljal.